README.md

```diff
@@ -162,8 +162,12 @@ If the user authorization fails, a `CanCan::AccessDenied` exception will be rais
 ```ruby
 class ApplicationController < ActionController::Base
   rescue_from CanCan::AccessDenied do |exception|
-    redirect_to root_url, :alert => exception.message
-  end
+      respond_to do |format|
+        format.json { head :forbidden, content_type: "text/html" }
+        format.html { redirect_to main_app.root_url, :notice => exception.message }
+        format.js   { head :forbidden, content_type: "text/html"}
```

Comment:
I like the idea of giving an example of all possible formats. Please prefer single quotes over double quotes when  they are not necessary: `"text/html"` --> `'text/html'` and use new hash rocket syntax. `:notice => exception.message` --> `notice: exception.message`